### ISSUE TITLE
Post title links require double taps on mobile

### ISSUE THREAD

**Issue by johnotander** [johnotander/pixyll#186]
https://twitter.com/4lpine/status/633382858422259712

**Comment by tgaeta** [comment]
I too am getting this. Happens on the `site.title` and the `post.title`.

**Comment by tgaeta** [comment]
Yup.. this fixed it for me. Commit showing the fix.. can't make a PR right now: https://github.com/tgaeta/tg-blog/commit/a0a1eff13c8d42c91110f775f50d618bf990b257
